scripts/generate_test_tsv.py

```diff
@@ -0,0 +1,206 @@
+                ('CollectionSite', 'Environment'), 
+                ('CollectionSite', 'Latitude'), 
+                ('CollectionSite', 'Longitude'), 
+                ('CollectionSite', 'Material'), 
+                ('Experiment', 'ExperimentName'), 
+                ('Lab', 'ContactEmail'), 
+                ('Lab', 'ContactName'), 
+                ('Lab', 'PrimaryInvestigator'), 
+                ('RawData', 'BarcodeSequence'), 
+                ('RawData', 'LinkerPrimerSequence'), 
+                ('RawData', 'RawDataID'), 
+                ('RawData', 'RawDataNotes'), 
+                ('RawDataProtocol', 'RawDataDatePerformed'), 
+                ('RawDataProtocol', 'RawDataProcessor'), 
+                ('RawDataProtocol', 'RawDataProtocolID'), 
+                ('RawDataProtocols', 'FinishingStrategyCoverage'), 
+                ('RawDataProtocols', 'FinishingStrategyStatus'), 
+                ('RawDataProtocols', 'NumberOfContigs'), 
+                ('RawDataProtocols', 'Primer'), 
+                ('RawDataProtocols', 'RawDataConditions'), 
+                ('RawDataProtocols', 'SequencingMethod'), 
+                ('RawDataProtocols', 'TargetGene'), 
+                ('Results', 'ResultID'), 
+                ('Results', 'ResultsLocation'), 
+                ('ResultsProtocol', 'ResultsDatePerformed'), 
+                ('ResultsProtocol', 'ResultsProcessor'), 
+                ('ResultsProtocol', 'ResultsProtocolID'), 
+                ('ResultsProtocols', 'ResultsMethod'), 
+                ('ResultsProtocols', 'ResultsTool'), 
+                ('ResultsProtocols', 'ResultsToolVersion'), 
+                ('Sample', 'SampleID'),
+                ('Sample', 'SampleWeight'),
+                ('Sample', 'SampleWeightUnit'),
+                ('SampleProtocol', 'SampleDatePerformed'),
+                ('SampleProtocol', 'SampleProcessor'),
+                ('SampleProtocol', 'SampleProtocolID'),
+                ('SampleProtocol', 'SampleProtocolNotes'),
+                ('SampleProtocols', 'SampleConditions'),
+                ('SampleProtocols', 'SampleTool'),
+                ('SampleProtocols', 'SampleToolVersion'),
+                ('Specimen', 'SpecimenCollectionDate'),
+                ('Specimen', 'SpecimenCollectionMethod'),
+                ('Specimen', 'SpecimenCollectionTime'),
+                ('Specimen', 'SpecimenID'),
+                ('Specimen', 'SpecimenNotes'),
+                ('Specimen', 'SpecimenWeight'),
+                ('Specimen', 'SpecimenWeightUnit'),
+                ('StorageLocation', 'StorageInstitution'),
+                ('StorageLocation', 'StorageFreezer'),
+                ('Study', 'RelevantLinks'),
+                ('Study', 'StudyName'),
+                ('Study', 'StudyType'),
+                ('Type', 'SpecimenType'),
+                ('Type', 'UberonCodeType')]
+
+    
+    # Blank data columns
+    headerCount = 3
+    diff = len(df)-headerCount
```

Comment:
```suggestion
    diff = len(df) - headerCount
```
Spaces around operator